bears/python/BanditBear.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='bandit')
+class BanditBear:
+    """
+    Performs security analysis on Python source code, utilizing the ``ast``
+    module from the Python standard library.
+    """
+    LANGUAGES = {"Python", "Python 2", "Python 3"}
+    REQUIREMENTS = {PipRequirement('bandit', '1.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Security'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         bandit_skipped_tests: typed_list(str)=
+                         ('B105', 'B106', 'B107', 'B404', 'B603', 'B606',
+                          'B607')):
+        """
+        :param bandit_skipped_tests:
+            The IDs of the tests ``bandit`` shall not perform. You can get
+            information about the available builtin codes at
+            https://github.com/openstack/bandit#usage.
```

Comment:
so on long term we'll have to make options out of those anyway, i.e. sane options. Where the user doesn't have to read up on linter specific error codes....